docs/software/file_servers.md

```diff
@@ -2,13 +2,171 @@
+
+    ![DietPi software](../assets/images/dietpi-software.jpg)
+
+    To see all the DietPi configurations options, review [DietPi Tools](../../dietpi_tools) section.
 
 [Return to the **Optimised Software list**](../../dietpi_optimised_software)
+
+## ProFTPD - Simple, efficient, lightweight FTP file server
+
+ProFTPD gives you access to files/music/downloads etc. on your DietPi system quickly and efficiently with minimal overhead.
+
+![DietPi file server software ProFTPD](../assets/images/dietpi-software-fileservers-proftpd.png)
+
+### Access to ProFTPD
+
+The access to ProFTPD with **Windows** is achieved as follows:
+
+- Go to My Computer (Windows Explorer).
+- In the address bar at the top type `ftp://dietpi:dietpi@192.168.0.100` press enter.
+
+Change 192.168.0.100 to the IP address of your DietPi system.
+
+The access to ProFTPD with an **FTP client** is achieved as follows:
+
+- username = dietpi (v6.2 and lower = root)
+- password = The same as your root login password. Default is dietpi
+- address = Your IP address (e.g.: 192.168.0.100)
+- port = 21
+
+### Options
+
+#### Destination directory
+
+The destination directory can be changed by replacing **/Path/To/Directory** to your target directory:
+
+```bash
+systemctl stop proftpd
+sed -i '/DefaultRoot /c\DefaultRoot /Path/To/Directory' /etc/proftpd/proftpd.conf
+systemctl start proftpd
+```
+
+#### Jailing (lock users to their home folders)
+
+*Jailing* can be enabled via
+
+```bash
+systemctl stop proftpd
+sed -i "/DefaultRoot /c\DefaultRoot ~" /etc/proftpd/proftpd.conf
+systemctl restart proftpd
+```
+
+See also <https://en.wikipedia.org/wiki/ProFTPD>.
+
+## Samba - Feature rich file server
+
+The Samba server lets you share files on your DietPi system with ease based on the well known SMB networking protocol.
+
+![DietPi file server software samba](../assets/images/dietpi-software-fileservers-samba.png)
+
+### Access to Samba
+
+The access to the Samba file server is achieved as follows:
+
+- Address = \\\192.168.0.100\dietpi or \\\dietpi\dietpi
+- username = dietpi (DietPi v6.12 and lower = root)
+- password = <your global application password\>
+
+### Options
+
+#### Change the Samba password
+
+The Samba password can be changed with
+
+```bash
+smbpasswd -a dietpi
+```
+
+#### Add/change the valid user
+
+To add/change the valid user do the following steps:
+
+- Edit /etc/samba/smb.conf
+    - Search the entry [dietpi], change `valid users = username_i_require`
+    - Add the user to Samba with `smbpasswd -a username_i_require`
+- Restart services with `dietpi-services restart`
+
+You can now connect to the samba server with the username and password you entered above.
+
+#### Destination directory
+
+The destination directory can be changed by replacing **/Path/To/Directory** to your target directory:
+
+```bash
+sed -i '/path = /c\path = /Path/To/Directory' /etc/samba/smb.conf
+dietpi-services restart
```

Comment:
No need to restart each and every service, lets do it simple and natively:
```
systemctl restart nmbd smbd
```

Reply:
Changed.